install_files/securedrop-keyring/DEBIAN/preinst

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+# Solution adapted from DKG's work on `deb.torproject.org-keyring`.
+# In SecureDrop versions before 0.3.10, the Ansible playbooks used
+# the apt_key module to add the signing key to /etc/apt/trusted.gpg.
+# It's cleaner to use the trusted.gpg.d subdirectory, since we can
+# update that trivially in future versions of the keyring package.
+#
+# Therefore let's clean up prior versions of the key installed
+# to the general apt keyring, to ensure we only have one signing key
+# installed for authenticating securedrop-related packages.
+
+if [ -e /etc/apt/trusted.gpg ] && which gpg >/dev/null; then
+   (
```

Comment:
These parens are unnecessary and hurt readability in my opinion because at first you wonder if they're doing anything.